mu-plugins/central-hub/tests/phpunit/unit/config-store/loadConfig.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ *  Tests for external API function loadConfig()
+ *
+ * @package    spiralWebDb\centralHub\Tests\Unit\ConfigStore
+ * @since      1.3.0
+ * @author     Robert A. Gadon
+ * @link       https://github.com/rgadon107/cornerstone
+ * @license    GNU General Public License 2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Unit\ConfigStore;
+
+use Brain\Monkey;
+use function KnowTheCode\ConfigStore\loadConfig;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_LoadConfig
+ *
+ * @package spiralWebDb\centralHub\Tests\Unit\ConfigStore
+ * @group   config-store
+ */
+class Tests_LoadConfig extends Test_Case {
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once CENTRAL_HUB_ROOT_DIR . '/src/config-store/api.php';
+	}
+
+	/**
+	 * Test loadConfig() should store a config given a valid store key and config.
+	 */
+	public function test_should_store_config_in_the_store_given_valid_key_and_config() {
+		$config = [
+			'aaa' => 'bbb',
+			'ccc' => 'ddd',
+		];
+		Monkey\Functions\expect( 'KnowTheCode\ConfigStore\_the_store' )
+			->once()
+			->with( 'foo', $config )
+			->andReturn( true );
+
```

Comment:
Be consistent with the code formatting.  Notice here you have 1 empty line but below there's none.  Be consistent.  Pick one style and stick with it.